src/strategies/api-post/index.ts

```diff
@@ -0,0 +1,38 @@
+import fetch from 'cross-fetch';
+import { formatUnits } from '@ethersproject/units';
+
+export const author = 'polychainmonsters';
```

Comment:
This should be a valid Github account
```suggestion
export const author = 'miertschink';
```